build.rs

```diff
@@ -24,7 +24,7 @@
 //! - [x] Linux 32 bit
 //! - [x] Linux 64 bit
 //! - [ ] MacOS 32 bit
```

Comment:
I don't have a 32 bit Apple machine, and so I have no way to test it.

Reply:
Last year Apple made 32-bit machines was 2011, so I think it is safe to remove this line.
```suggestion

```